### ISSUE TITLE
Contrast issues

### ISSUE THREAD

**Issue by phil294** [manekinekko/windows-nt-vscode-theme#3]
Hi! Thanks for this extension. I noticed that in the search bar, when you select text, it is not readable anymore:

![image](https://waritschlager.de/share/A90330EED1F5FB4864C3A59DB39DAEB4.png)

Text color should probably be white. I quickly looked into this and it seems that there is **no** such setting as `selection.foreground`?? So is there even any way to solve this? If not, I'd suggest setting `selection.background` to a light grey color instead the dark blue :-/

Another issue: The action buttons on the top right are in the wrong color:

![image](https://waritschlager.de/share/FB1FD4181DD1475B8876258B6A93EEAE.png)

There are actually multiple icons in the top row, but you can barely see them. (GitLens icons, mostly). I also looked into this: Currently, dev tools say

```css
.hc-black .monaco-action-bar .action-item.menu-entry .action-label, .vs-dark .monaco-action-bar .action-item.menu-entry .action-label {
    background-image: var(--menu-entry-icon-dark);
}
```
If you change this to `--meny-entry-icon-light`, then it gets readable:
![image](https://waritschlager.de/share/62951E82BC7D2FACD9C54A15F20D8BB1.png)

VSCode seems to apply these flags based on the fact that win-nt-theme is marked as a dark theme. Can this be changed to be a light theme? Would also make more sense imo.

**Comment by phil294** [comment]
The issue for point 2 seems to be in package.json `contributes.themes.uiTheme` which needs to be `"vs"` instead of `"vs-dark"`. https://github.com/microsoft/vscode/issues/140006, https://github.com/slack-theme/visual-studio-code/issues/9#issuecomment-501522227, https://code.visualstudio.com/api/references/contribution-points#theme-example.
Cannot find any documentation about this though...

**Comment by manekinekko** [comment]
Hey, thank you for looking at this. Do you want to send a PR to fix this issue?

**Comment by phil294** [comment]
> Hey, thank you for looking at this. Do you want to send a PR to fix this issue?

sure! #4

don't know what to do about the selection color issue though

**Comment by manekinekko** [comment]
Closed by #4 and https://github.com/manekinekko/windows-nt-vscode-theme/commit/6539d31af6ef9e1e58ba1f303e7d41e58d47c4e7
